core/src/test/scala/org/apache/spark/ui/UISeleniumSuite.scala

```diff
@@ -44,7 +46,33 @@ class UISeleniumSuite extends FunSuite with WebBrowser with Matchers with Before
   implicit var webDriver: WebDriver = _
 
   override def beforeAll(): Unit = {
-    webDriver = new HtmlUnitDriver
+    webDriver = new HtmlUnitDriver {
+
+      getWebClient.setCssErrorHandler(new DefaultCssErrorHandler {
```

Comment:
We can put the test class into core then have the other projects declare a test-jar test dependency on core.  I think that SQL uses this pattern to share test code between SQL core and catalyst.